[card.io] moved to open source

Redth opened the pull request:
Moving the latest version of Card.IO bindings for iOS and Android into the open source repository.

Redth committed:
[card.io] moved to open source

bholmes commented on XPlat/CardIO/Android/samples/CardIOSampleAndroid/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,23 @@
+[assembly: AssemblyConfiguration ("")]
+[assembly: AssemblyCompany ("")]
+[assembly: AssemblyProduct ("")]
+[assembly: AssemblyCopyright ("redth")]
```
Make Xamarin Inc.

bholmes commented on XPlat/CardIO/Android/source/CardIO.Android/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,23 @@
+[assembly: AssemblyConfiguration ("")]
+[assembly: AssemblyCompany ("")]
+[assembly: AssemblyProduct ("")]
+[assembly: AssemblyCopyright ("redth")]
```
Make Xamarin Inc

bholmes commented on XPlat/CardIO/iOS/samples/CardIOSampleiOS/Main.cs:
```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+
+#if __UNIFIED__
```
Pull Classic code and builds

Redth committed:
[card.io] Fix assembly attributes

Redth committed:
[card.io] remove some traces of iOS classic api builds
Classic API has not been supported in this component for several releases, however there were remnants of this support that needed cleaning up.

bholmes commented:
iOS does not build.  I think you are missing the externals folder.

`xcodebuild: error: './externals/CardIOSharp/CardIOSharp.xcodeproj' does not exist.`

Redth committed:
[card.io] add missing XCode project from externals
